pkg/domain/entities/images.go

```diff
@@ -306,6 +307,30 @@ type ImageSaveOptions struct {
 	Quiet bool
 }
 
+// ImageScpOptions provide options for securely copying images to podman remote
+type ImageScpOptions struct {
+	// DestImageName is the image the user desires to load on their machine or remote machine
+	DestImageName string
```

Comment:
If this is never used, it can be removed. Maybe turn this into a comment in `parseArgs` like “in the future, this function could, instead of rejecting renames, also set a DestImageName field” to record the intent.